test/utils/stubs.js

```diff
@@ -6,12 +6,18 @@
 const underscore = require('underscore');
 
 module.exports = {
+  getNorthstarAPIBaseUri: function getNorthstarAPIBaseUri() {
+    return process.env.DS_NORTHSTAR_API_BASEURI || 'https://northstar-thor.dosomething.org/v1';
```

Comment:
Might be better to default to localhost instead.

Reply:
I've never user a local Northstar and the `.env.example` uses `https://northstar-fake.dosomething.org/v1` should I use that one?

Reply:
Yep. The idea is you could use a lib that would fake responses from this domain. Can be really helpful with tests.

Reply:
Ok. I use nock for tests. Updated.